### PR TITLE
Revert "[FVR-190] Puma - Add request queue depth"

### PR DESCRIPTION
**Pull request by indiebrain**
Reverts powerhome/APP-Grafana-Dashboards#68

Part of: https://github.com/powerhome/nitro-web/pull/39489

This complexity was introduced into nitro-web to measure and chart Puma Request Queue depth. Originally we were trying to gather context on elevated 5xx HTTP responses when switching webservers to Puma in nitro-web-production. Observations in https://github.com/powerhome/nitro-web/pull/39394 illustrate that, in practice, this measure is not as interesting as I'd originally assumed.

### CODE REVIEW

File: puma.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "puma_queued_requests{namespace=~\"$namespace\",job=~\".*$job.*\",pod=~\".*$filter.*\",pod=~\"$pod\"}",
+          "expr": "puma_backlog{namespace=~\"$namespace\",job=~\".*$job.*\",pod=~\".*$filter.*\",pod=~\"$pod\"}",
```
Comment (benlangfeld) [inline puma.json:71]: Do we want the backlog size at all or should we just remove this pane entirely?

Comment (indiebrain) [reply puma.json:71]: I'm not married to it, but I'd like to keep it for now. 

TLDR: Sounds like it _could_ be useful as applications scale, and it's low cost.

We've not had a great understanding of this stat's semantics, we never saw it fluctuate so we removed it from the dashboard. However, the puma stats documentation indicates that this could be useful in determining if the Application Server topology needs to change due to increases in client demand. 

> if this is above 0, you need more capacity -- https://puma.io/puma/file.stats.html

It's already provided by default by `pumactl` application, and the measure is automatically exposed by the Puma Prometheus exporter.

**Review by guilhermeoki — APPROVED** [review]
